front/lib/front_web/templates/layout/_license_banner.html.eex

```diff
@@ -0,0 +1,20 @@
+<!-- Only show banner if license is invalid in EE version -->
+<%= if Front.ee?() do %>
+  <%= if not @license_status.valid do %>
+    <div id="license-expired-banner" style="position: fixed; top: 0; left: 0; width: 100%; background-color: #d32f2f; color: white; text-align: center; padding: 10px 0; font-weight: bold; z-index: 10000;">
+      Your EE license is not valid
```

Comment:
The users will probably not know what EE is short for.
Let's change it to a more descriptive message with a call to action link, for example:

`You are running a Semaphore Enterprise Edition server without a valid license. 
Some features will be limited and new commits will not trigger workflows.
You can find more information about getting a license here[doc_link].